src/resolver.ts

```diff
@@ -56,6 +59,7 @@ export const resolver = <T, Chain, R>(wretch: T & Wretch<T, Chain, R>) => {
         err["cause"] = referenceError
         err.stack = err.stack + "\nCAUSE: " + referenceError.stack
         err.response = response
+        err.request = new Request(finalUrl, finalRequestInit)
```

Comment:
The current major version of `wretch` is compatible with node v14, so using `Request` without polyfill would break the contract.